pages/index.tsx

```diff
@@ -48,35 +47,53 @@ const BugList = ({bugs}: {bugs: any[]}) => {
 
 
 const Home: NextPage = (props) => {
+  const [winReady, setwinReady] = useState(false);
+  useEffect(() => {
+      setwinReady(true);
+  }, []);
 
-  const [state, setState] = useState({bugs: props.data});
+  const [state, setState] = useState(props.data);
 
-  
-  
   function onDragEnd(result: any) {
-    console.log({result, state})
+    console.log({state})
     if (!result.destination) {
       return;
     }
 
     if (result.destination.index === result.source.index) {
       return;
     }
-
+    
     if (result.destination.droppableId === result.source.droppableId) {
```

Comment:
Checks to see if destination and source are the same and if so, apply the same single column logic that we've been using.